content/guides/composite_monitors.md

```diff
@@ -131,6 +131,21 @@ Consider a composite monitor that uses three individual monitors—A, B, and C
 
 Two of the four scenarios will trigger an alert, even though not all of the individual monitors have the most severe status, `Alert` (and in row 1, none do). But how _many_ alerts might you potentially receive from the composite monitor? That depends on the individual monitors' alert types.
 
+### How a component monitor's status is determined
```

Comment:
`composite` ?

Reply:
I think `component` is correct? I'm explaining how their status is determined. Open to suggestions for different wording. Maybe:

    ### How a composite monitor's component monitors' status is determined

But that's kinda long-winded

Reply:
oh i see it makes sense. forget my comment :)